variables.tf

```diff
@@ -260,6 +260,16 @@ variable "kube_exec_auth_aws_profile_enabled" {
   description = "If `true`, pass `kube_exec_auth_aws_profile` as the `profile` to `aws eks get-token`"
 }
 
+variable "kube_exec_auth_api_version" {
+  type        = string
+  default     = "client.authentication.k8s.io/v1alpha1"
```

Comment:
Perhaps it makes sense to just take it from a predefined map in `locals` based on a `variable "kubernetes_version"`?

Reply:
It isn't tied to kubernetes version, instead it should match API version returned by `aws eks get-token`. See linked PR, this was changed recently. I didn't find any sensible way to autodetect this in module, this value should be known before making exec call.